### PR TITLE
Prepare for 2.0 release

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

**Comment by garyb** [comment]
Oh, actually, there was the newtype iso thing that we could include here too, right?

File: src/Data/Profunctor/Costar.purs [hunk 1]
```diff
@@ -43,12 +42,12 @@ instance applicativeCostar :: Applicative (Costar f a) where
...
 
 instance bindCostar :: Bind (Costar f a) where
-  bind (Costar m) f = Costar \x -> unCostar (f (m x)) x
+  bind (Costar m) f = Costar \x -> case f (m x) of Costar g -> g x
 
 instance monadCostar :: Monad (Costar f a)
 
 instance distributiveCostar :: Distributive f => Distributive (Costar f a) where
-  distribute f = Costar \g -> map ((_ $ g) <<< unCostar) f
+  distribute f = Costar \a -> map (\(Costar g) -> g a) f
```
Comment (paf31) [inline src/Data/Profunctor/Costar.purs:50]: Do we need the `Distributive` constraint on `f` here?

**Comment by paf31** [comment]
Looks good. Yes, having something like

``` purescript
mapNewtype :: forall new old p. (Newtype new old, Profunctor p) => p old old -> p new new
```

would be good, but we can leave it for a minor release if you like.

File: src/Data/Profunctor.purs [hunk 2]
```diff
@@ -33,5 +34,11 @@ rmap b2c = dimap id b2c
 arr :: forall a b p. (Category p, Profunctor p) => (a -> b) -> p a b
 arr f = rmap f id
 
+wrapIso :: forall t a p. (Newtype t a, Profunctor p) => p t t -> p a a
+wrapIso = dimap wrap unwrap
+
+unwrapIso :: forall t a p. (Newtype t a, Profunctor p) => p a a -> p t t
```
Comment (paf31) [inline src/Data/Profunctor.purs]: Do you think we need a `a -> t` argument here to make type inference work in some cases?

Comment (garyb) [reply src/Data/Profunctor.purs]: Ah good point, probably. I was going to ask though - do we need both versions here? I know you can turn a profunctor iso lens around, but you need to bring lenses in for that, right?

Comment (paf31) [reply src/Data/Profunctor.purs]: I think both are potentially useful, right? Actually, are these the wrong way around?

Comment (garyb) [reply src/Data/Profunctor.purs]: Only if you think the names should make sense... ahem.

Comment (garyb) [reply src/Data/Profunctor.purs]: On that note, are these names any good anyway?

Comment (paf31) [reply src/Data/Profunctor.purs]: Yeah I think they're good.

**Pull request merged**